src/main/java/com/client/core/base/tools/web/impl/StandardJsonConverter.java

```diff
@@ -1,25 +1,24 @@
 package com.client.core.base.tools.web.impl;
 
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
-
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
-import org.springframework.stereotype.Service;
-
 import com.client.core.base.tools.web.JsonConverter;
 import com.client.core.base.tools.web.objectmapper.CustomJsonObjectMapper;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.JavaType;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.collect.Maps;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
 
 @Service
 public class StandardJsonConverter implements JsonConverter {
 
-	private static Log log = LogFactory.getLog(StandardJsonConverter.class);
+	private static final Log log = LogFactory.getLog(StandardJsonConverter.class);
```

Comment:
Use Lombok

Reply:
I just moved all loggers to use `@Log4j2` annotation